controller/commentsController.js

```diff
@@ -405,3 +404,40 @@ exports.getCommentVotes = async (req, res, next) => {
     );
   }
 };
+exports.getSingleComment = async (req, res, next) => {
+  const { refId } = req.query;
```

Comment:
Please remove this. Getting a single comment only needs the commentId from the url params. RefId is not used as this could return several records defeating the purpose of this controller.

Reply:
@dave-ok please revert this merge..